package config;

message TransferInfo {
	optional string listen_ip = 1;
	optional int32 listen_port = 2;
	optional int32 number = 3;
	optional int32 server_zone = 4;
}

message TransferConfig {
	repeated TransferInfo transfer_list = 1;
}